client/src/views/UserProfile.vue

```diff
@@ -149,7 +143,11 @@ const props = defineProps({
 // TODO: Warn user if there are unsaved changes --> Use "beforeRouteLeave lifecycle hook"
 const { fetchProfile, userData } = useUserProfile();
 
-fetchProfile();
+try {
```

Comment:
Warum kein try-catch im composable?

Reply:
in composable code von anderen scheint es best practice zu sein das im parent zu machen. dann kann das parent auch entscheiden was mit dem Fehler gemacht wird. comosables sollen ja unkompliziert und universell einsetzbar sein.

Reply:
Ja, klingt vernüftig